.github/workflows/deploy.yaml

```diff
@@ -4,6 +4,7 @@ on:
   pull_request:
     branches:
       - "package-*"
+      - develop
```

Comment:
What is now happening (compared to previously) when we get a pull request against develop?

Reply:
This is just a test thing for this PR, I'm removing it before merge.